tenacity/before_sleep.py

```diff
@@ -36,18 +36,27 @@ def before_sleep_log(
     """Before call strategy that logs to some logger the attempt."""
 
     def log_it(retry_state: "RetryCallState") -> None:
+        if retry_state.outcome is None:
+            return None
+
         if retry_state.outcome.failed:
             ex = retry_state.outcome.exception()
             verb, value = "raised", f"{ex.__class__.__name__}: {ex}"
 
             if exc_info:
-                local_exc_info = retry_state.outcome.exception()
+                local_exc_info = retry_state.outcome.exception() is not None
```

Comment:
This change breaks the existing test (`TestBeforeAfterAttempts.test_before_sleep_log_raises_with_exc_info`).
The `local_exc_info` is passed to `logger.log` so it should not be bool. 
Instead, the value should be the exception itself or None.
https://docs.python.org/3/library/logging.html#logging.LogRecord

Reply:
@ysk24ok Thanks for your comments! I'll take a look tonight.